addons/dragging/XEH_postInit.sqf

```diff
@@ -1,9 +1,72 @@
 // by PabstMirror, commy2
 #include "script_component.hpp"
 
+[QGVAR(moveCorpse), {
+    params ["_corpse", "_dir", "_posATL"];
+    _posATL params ["_xPos", "_yPos", "_zPos"];
+
+    if (isNull _corpse) exitWith {};
+
+    private _currentPos = getPosATL _corpse;
+
+    // Check if the corpse is already close to the target
+    // If so, don't teleport
+    if !(
+        (_currentPos select 0 <= _xPos + 0.25) &&
+        {_currentPos select 0 >= _xPos - 0.25} &&
+        {_currentPos select 1 <= _yPos + 0.25} &&
+        {_currentPos select 1 >= _yPos - 0.25} &&
+        {_currentPos select 2 <= _zPos + 0.25} &&
+        {_currentPos select 2 >= _zPos - 0.25}
```

Comment:
ever heard of distance?

```suggestion
_currentPos distance _posATL > 0.25
```

Reply:
It's not the same, but there's no specific reason why the volume to be checked should be a cube and not a sphere - it's just the first thing I thought of.